cionic/dsp.py

```diff
@@ -59,81 +72,86 @@ def normalize_fft(fft_data, freq):
 
 def calculate_cdf(fft_data, freq):
     """
-        Calculate normalized CDF of Welch Density FFT
-            data = numpy array or pandas series
-            NOTE: len(cdf) = len(data) - 1  Make sure to account for this in plotting plot(freq[1::], cdf)
+    Calculate normalized CDF of Welch Density FFT
+        data = numpy array or pandas series
+        NOTE: len(cdf) = len(data) - 1  Make sure to account for this in plotting
+        plot(freq[1::], cdf)
     """
     norm_data = normalize_fft(fft_data, freq)
     cdf = integrate.cumtrapz(norm_data, freq)
     return cdf
 
 
 def median_freq(rawEMGPowerSpectrum, frequencies):
-    """ From 'pysiology' module (can't import module directly because matplotlib.pyplot has compatiblity issues with pyqtgraph)
-        https://pypi.org/project/pysiology/
+    """From 'pysiology' module (can't import module directly because matplotlib.pyplot
```

Comment:
nit: should there be a newline before `From`? also, I think `pysiology` is spelled incorrectly